test/serializer/abstract/Switch.js

```diff
@@ -0,0 +1,11 @@
+let x = global.__abstract ? __abstract("number", "1") : 1;
```

Comment:
Add this as the first line of the test:
// throws introspection error

Reply:
Nice. And so I learn how the test framework goes...;-)